### PR TITLE
CAT-453 Create database for risk profiler and enable JPA

### PR DESCRIPTION
**Pull request by steverendell**
Also create skeleton batch job, WIP

More tests to follow

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/Status.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.riskprofiler.model;
+
+public enum Status {
+    NEW, ADDED_TODO, IGNORED
```
Comment (Mjwillis) [inline src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/Status.java]: Are all these status used?  Can only see NEW

File: src/main/resources/db/migration/V1_2__prison_supported.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS PRISON_SUPPORTED;
+
+CREATE TABLE PRISON_SUPPORTED
```
Comment (Mjwillis) [inline src/main/resources/db/migration/V1_2__prison_supported.sql]: Are you going to manually populate these migrated ones?  Might be nice to have an endpoint to populates maybe with SYSTEM_USER as protection?

Comment (steverendell) [reply src/main/resources/db/migration/V1_2__prison_supported.sql]: Good idea, can add this in next iteration. I was thinking manually populate to begin with.

File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/Status.java (hunk 1 above)
Comment (steverendell) [reply src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/Status.java]: Not yet, theyre just a guess at what will be needed

**Pull request merged**